enterprise/frontend/src/embedding-sdk/cli/snippets/analytics-css-snippet.ts

```diff
@@ -36,14 +44,15 @@ export const ANALYTICS_CSS_SNIPPET = `
 
 .analytics-header-right > a {
   color: #509EE3;
+  text-decoration: none;
 }
 
 .dashboard-select {
   background: transparent;
   color: #509EE3;
   border: none;
   font-family: inherit;
-  font-size: 14px;
+  font-size: 16px;
```

Comment:
This makes the `<select>` the same font size as other links.

![CleanShot 2567-11-08 at 01 13 05@2x](https://github.com/user-attachments/assets/8be9ab0f-d841-4679-a8c5-44405c3e2f61)